frontend/src/features/contacts/contact/detail/Container.tsx

```diff
@@ -0,0 +1,33 @@
+import { useContactDetail } from 'features/contacts/hooks/useContactDetail';
+import * as React from 'react';
+import styled from 'styled-components';
+
+import { ContactBreadcrumb } from '../..';
+import * as Styled from '../../styles';
+import PersonView from './Person';
+
+interface IContactViewContainerProps {
+  match?: any;
+}
+
+const ContactViewContainer: React.FunctionComponent<IContactViewContainerProps> = props => {
```

Comment:
Best practice is to have filename match the name of the main exported component. I would expect to export Contact here. If you don't like having multiple imports for "Contact" then please rename the corresponding files?

Also, this is fine (but I get why it would be confusing)

- contact/details/Container (exports Container)
- contacts/create/Container (exports Container)

Somewhere else, say **AppRouter.tsx**:

```tsx

import Container as ViewContainer from "contact/details";
import Container as CreateContainer from "contacts/create";
import Container as UpdateContainer from "contacts/edit";

// Use as ViewContainer, CreateContainer and UpdateContainer...
```